### ISSUE TITLE
GAHI question dropbox shows error message on new application

### ISSUE THREAD

**Issue by TiCampbell** [Get-Your/Get-Your#84]
For new applications in Step 3 (`finances`), the dropdown for gross annual household income displays an error message on page load. This is the error message that's expected to pop up if the user enters a non-integer into the 'how many individuals...?' input (but not until then), so this issue is probably related to the new 'update information' workflow, where the dropdown function is expecting a value even though `update_mode`==False.

Reproducible in Firefox 111.0.1 and Chrome 111.0.5563.147 on Windows 10.

![Image](https://user-images.githubusercontent.com/44819630/230736025-cac5e005-2592-43c9-b622-65c7f2da4fde.png)

**Comment by TiCampbell** [comment]
The GAHI dropdown has been removed for v2, so this is no longer relevant.
